packages/airnode-deployer/src/infrastructure/gcp.ts

```diff
@@ -1,22 +1,183 @@
 import { GcpCloudProvider } from '@api3/airnode-node';
+import { go } from '@api3/promise-utils';
 import { Storage } from '@google-cloud/storage';
+import {
+  BUCKET_NAME_REGEX,
+  Directory,
+  FileSystemType,
+  generateBucketName,
+  translatePathsToDirectoryStructure,
+} from '../utils/infrastructure';
 import * as logger from '../utils/logger';
 
-export async function stateExists(bucketName: string, cloudProvider: GcpCloudProvider) {
-  logger.debug('Checking Terraform state existence in GCP');
-  const { projectId } = cloudProvider;
-  const storage = new Storage({ projectId });
-  const bucket = storage.bucket(bucketName);
+const initializeGCSService = (cloudProvider: GcpCloudProvider) => new Storage({ projectId: cloudProvider.projectId });
```

Comment:
Our concention is to uppercase just the first letter even for all-uppercase words (e.g. HTTP is Http, GCP should be Gcp, so GCS should be Gcs).